apps/a11y-tests/src/tests/__snapshots__/ComponentExamples.test.tsx.snap

```diff
@@ -399,151 +399,11 @@ Array [
-    },
-    "properties": Object {
-      "tags": Array [
-        "Accessibility",
-      ],
-    },
-    "ruleId": "color-contrast",
-  },
-  Object {
-    "level": "error",
-    "locations": Array [
-      Object {
-        "annotations": Array [
-          Object {
-            "snippet": Object {
-              "text": "<span style=\\"padding-bottom:1px\\">https://</span>",
-            },
-          },
-        ],
-        "fullyQualifiedLogicalName": ".ms-TextField.root-3:nth-child(2) > .ms-TextField-wrapper.wrapper-4 > .fieldGroup-5.ms-TextField-fieldGroup > .prefix-10.ms-TextField-prefix > span",
-        "physicalLocation": Object {
-          "fileLocation": Object {
-            "uri": "about:blank",
-          },
-        },
-      },
-    ],
-    "message": Object {
-      "richText": "Fix any of the following:
-- Element has insufficient color contrast of 2.43 (foreground color: #a6a6a6, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1",
-      "text": "Fix any of the following: Element has insufficient color contrast of 2.43 (foreground color: #a6a6a6, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1.",
-    },
-    "partialFingerprints": Object {
-      "fullyQualifiedLogicalName": ".ms-TextField.root-3:nth-child(2) > .ms-TextField-wrapper.wrapper-4 > .fieldGroup-5.ms-TextField-fieldGroup > .prefix-10.ms-TextField-prefix > span",
-      "ruleId": "color-contrast",
-    },
-    "properties": Object {
-      "tags": Array [
-        "Accessibility",
-      ],
-    },
-    "ruleId": "color-contrast",
-  },
-  Object {
-    "level": "error",
-    "locations": Array [
-      Object {
-        "annotations": Array [
-          Object {
-            "snippet": Object {
-              "text": "<span style=\\"padding-bottom:1px\\">.com</span>",
-            },
-          },
-        ],
-        "fullyQualifiedLogicalName": ".ms-TextField.root-3:nth-child(2) > .ms-TextField-wrapper.wrapper-4 > .fieldGroup-5.ms-TextField-fieldGroup > .ms-TextField-suffix.suffix-11 > span",
-        "physicalLocation": Object {
-          "fileLocation": Object {
-            "uri": "about:blank",
-          },
-        },
-      },
-    ],
-    "message": Object {
-      "richText": "Fix any of the following:
-- Element has insufficient color contrast of 2.43 (foreground color: #a6a6a6, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1",
-      "text": "Fix any of the following: Element has insufficient color contrast of 2.43 (foreground color: #a6a6a6, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1.",
-    },
-    "partialFingerprints": Object {
-      "fullyQualifiedLogicalName": ".ms-TextField.root-3:nth-child(2) > .ms-TextField-wrapper.wrapper-4 > .fieldGroup-5.ms-TextField-fieldGroup > .ms-TextField-suffix.suffix-11 > span",
```

Comment:
Hmm, this is a bit odd, I'm not sure I can explain this one based on the PR changes. @statm Do you have any ideas?

Reply:
@kenotron mentioned some missing a11y snapshot updates earlier?

Reply:
I see where my confusion is. Master was in a good state, but it appears a11y-tests are not running against master. Should they be?